### PR TITLE
Review "Task" page 

### PR DESCRIPTION
**Pull request by jdbaldry**
- Use unordered lists where they are semantically appropriate.
- Use full URLs.
- Update one out of date link ([Create a dashboard and add a panel](/docs/grafana/latest/dashboards/add-organize-panels/#create-a-dashboard-and-add-a-panel)).
- Use images uploaded to the asset service.
- Use the admonition shortcode.
- Use semantic line breaks.

Closes https://github.com/grafana/writers-toolkit/issues/544

Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

### CODE REVIEW

File: docs/sources/structure/topic-types/task/index.md [hunk 1]
```diff
@@ -20,35 +21,38 @@ Task topics include numbered steps that describe how to achieve an outcome.
...
+- **Steps:** Users are provided with a directive through numbered steps.
 
-- Write steps so that they contain one action, or possibly two related actions, such as _Copy and paste a value_ or _Save and quit the program._
-- Unless a sentence instructs the reader to act, it isn't a step.
+  Write steps so that they contain one action, or possibly two related actions, such as _Copy and paste a value_ or _Save and quit the program._
 
-<figure>
-<img src="task.png" alt="Task structure" width="600">
-</figure>
+  Unless a sentence instructs the reader to act, it isn't a step.
```
Comment (clayton-cornell) [inline docs/sources/structure/topic-types/task/index.md:53]: For me... this is a weird sounding sentence.

Could it be rewritten to say something like:

`A step must instruct the reader to act or perform a specific task.`

or something similar?

**Review by JStickler — APPROVED** [review]
LGTM

**Review by chri2547 — APPROVED** [review]
Just one slight adjustment that Clayton mentioned and good to go!

File: docs/sources/structure/topic-types/task/index.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/structure/topic-types/task/index.md:53]:
```suggestion
  A step must instruct the reader to perform a specific task.
```

**Pull request merged**